### PR TITLE
i18n(es): Translated `reading-time.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translated `reading-time.mdx`

### CODE REVIEW

File: src/content/docs/es/recipes/reading-time.mdx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+  ```js title="remark-reading-time.mjs"
+  import getReadingTime from 'reading-time';
+  import { toString } from 'mdast-util-to-string';
+
+  export function remarkReadingTime() {
+    return function (tree, { data }) {
+      const textOnPage = toString(tree);
+      const readingTime = getReadingTime(textOnPage);
+      // readingTime.text  nos dará el tiempo de lectura en minutos como un string amigable,
```
Comment (dreyfus92) [inline src/content/docs/es/recipes/reading-time.mdx:50]:
```suggestion
      // readingTime.text nos dará el tiempo de lectura en minutos como un string legible,
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Te dejo una sugerencia @Waxer59, buen trabajo 😁

**Comment by Waxer59** [comment]
Gracias por la revisión 🤠

File: src/content/docs/es/recipes/reading-time.mdx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+description: Construye un plugin de remark para añadir tiempo de lectura a tus archivos Markdown o MDX.
+i18nReady: true
+type: recipe
+---
+
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+Crea un plugin de [remark](https://github.com/remarkjs/remark) que añada una propiedad de tiempo de lectura al frontmatter de tus archivos Markdown o MDX. Utiliza esta propiedad para mostrar el tiempo de lectura de cada página.
+
+## Recipe
```
Comment (Yan-Thomas) [inline src/content/docs/es/recipes/reading-time.mdx:12]: Missing translation here!

Comment (Waxer59) [inline src/content/docs/es/recipes/reading-time.mdx:12]:
```suggestion
## Receta
```

**Review by Yan-Thomas — APPROVED** [review]
LGTM!

**Pull request merged**